### ISSUE TITLE
No way of changing convertUnknownCategoricalLevelsToNa using prediction-service-builder

### ISSUE THREAD

**Issue by ksbg** [h2oai/steam#377]
When manually loading a POJO model using Java, I can simply set _convertUnknownCategoricalLevelsToNa_ to _True_ when loading it, like:

```
EasyPredictModelWrapper model = new EasyPredictModelWrapper(
    new EasyPredictModelWrapper.Config()
        .setModel(rawModel)
        .setConvertUnknownCategoricalLevelsToNa(false));
```

However, as I understand it, this is not possible when using Steam's prediction service builder. There are no options or parameters that would allow this while building the .war file. So it's impossible to make predictions using unknown categorical labels when using the service builder.

**Comment by mstensmo** [comment]
Yes that is correct. However, this is open source so you can change the code to do what you want. See here: https://github.com/h2oai/steam/blob/master/prediction-service-builder/src/main/webapp/extra/src/ServletUtil-TEMPLATE.java#L73 and https://github.com/h2oai/steam/blob/master/prediction-service-builder/src/main/webapp/extra/src/ServletUtil-TEMPLATE.java#L91

**Comment by ksbg** [comment]
Yes I've already done that, however this seems like an important functionality, so I feel like adding this functionality (e.g. as an option parameter) would make sense. If people agree with me, I'm willing to add it.

**Comment by mstensmo** [comment]
The problem is that some people want the default functionality, with an error for unknown categorical values -- because then people sent the wrong data in. In other cases, like you, you want it to be NA. So to add it you need to send an additional parameter like you say, which makes things more complicated.
